README.md

```diff
@@ -44,6 +44,9 @@ run style checks use `composer cs`.
 
 ## Release notes
 
+### Next Version (2.0.x)
+* Ability ro re-ruse the file `cli-config.php` when including `FundraisingStore` in an application is removed.
```

Comment:
You are right. While changing the readme I thought "I should really add the headline 'Breaking changes'" and then promptly forgot about it :( . Although it seems silly to raise the full version number for such a small change, it is necessary. 

Should I resubmit the PR for a new 3.0 branch? The change is not that important and we can collect bugfixes and small improvements in 2.0.x